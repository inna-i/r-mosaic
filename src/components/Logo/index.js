import React from 'react';

function Logo() {
	return (
		<div className="w-logo">
			<div className="logo">Logo</div>
		</div>
	);
}

export default Logo;
