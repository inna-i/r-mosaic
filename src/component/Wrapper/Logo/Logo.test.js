import React from 'react';
import { shallow } from 'enzyme';

import Logo from '.';

describe('Logo  component', () => {
	it('should render Logo  component', () => {
		// when
		const wrapper = shallow(<Logo />);

		// then
		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
