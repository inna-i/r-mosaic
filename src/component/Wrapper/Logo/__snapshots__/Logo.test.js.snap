// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo  component should render Logo  component 1`] = `
<div
  className="w-logo"
>
  <div
    className="logo"
  >
    Logo
  </div>
</div>
`;
